### PR TITLE
add information about quickstart

### LINKED ISSUE #1989: Juju quickstart deprecation
we shoud probably add a note or page about this in 2.x - there are many extant blogs/askubuntu posts that reference quickstart, it still exists in the 16.04 archive, etc. If we had a mention we could at least hopefully garner more google-fu for it than an old blog, and explain that it is not required for 2.0

### PR DESCRIPTION
**Pull request by evilnick**
This is the disambiguation for quickstart. Intentionally not linked from the navigation.

Fixes #1989

### CODE REVIEW

File: src/en/juju-quickstart.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Title: Juju QuickStart
```
Comment (degville) [inline src/en/juju-quickstart.md]: Is the capital 'S' in *QuickStart* correct?

File: src/en/juju-quickstart.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# Juju Quickstart
+
+The ***quickstart*** utility was used with earlier versions (1.x) of Juju. It was designed to help users quickly deploy charms and bundles with those versions of Juju and allowed editing and management of various different Juju clouds.
+
+The functionality provided by `juju quickstart` is not supported or required in the current version of Juju. Most of this functionality (native support for bundles, interactive setup for clouds) is now included in Juju itself. If you encounter any charm documentation, blog posts or other online sources that specify the use of `quickstart` it is likely they refer to a bundle. Bundles can now be deployed directly by Juju in the same ways as charms. See the current documentation on ["Using and Creating Bundles"][bundles] for more detail.
+
```
Comment (degville) [inline src/en/juju-quickstart.md]: for more *details*?
(I'm not quite sure whether this is intentional or not).

File: src/en/juju-quickstart.md (hunk 1 above)
Comment (evilnick) [inline src/en/juju-quickstart.md]: no :)

File: src/en/juju-quickstart.md (hunk 2 above)
Comment (evilnick) [inline src/en/juju-quickstart.md]: it was but it is better with an s

**Pull request merged**